doc/news/_preparation_next_release.md

```diff
@@ -204,6 +204,7 @@ _(@tucek)_
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
+- Work on COMPILE.md and INSTALL.md to help with understanding _(@a-kraschitzer)_
```

Comment:
Thank you for the PR! Please fix the typos found by @mpranj (@mpranj thank you for the review!). Please also add links in the release notes (_preparation_next_release.md ).